app/policies/birth_record_policy.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class BirthRecordPolicy < ApplicationPolicy
+  def show?
+    true
+  end
+
+  def index?
```

Comment:
we're using policy scopes on index, so we probably don't need this

Reply:
true, go ahead and delete